analysis/neutron/openmc_model.py

```diff
@@ -173,6 +189,11 @@ def baby_geometry(x_c: float, y_c: float, z_c: float):
     gap_region = +z_plane_8 & -z_plane_9 & -z_cyl_1 & +right_cyl
     firebrick_region = +z_plane_5 & -z_plane_11 & +z_cyl_3 & -z_cyl_4
     heater_region = -right_cyl
+    table_under_source_region = +z_plane_15 & -z_plane_14 & -sphere
+    lead_block_1_region = -rectangular_1
+    lead_block_2_region = -rectangular_2
+    lead_block_3_region = -rectangular_3
+    lead_block_4_region = -rectangular_4
```

Comment:
So by reading everything I now understand that `rectangular_1` is for lead bricks. Do we even need `rectangular_1`? It seems that variable isn't used anywhere else. Could we not simply have:

```python
lead_block_1_region = openmc.model.RectangularParallelepiped(
        -17.5 + x_c, -9.5 + x_c, -8 + y_c, +8 + y_c, z_c - 28.00, z_c - 24.00
    )
lead_block_1_region = - lead_block_1_region
```